app/pb_kits/playbook/pb_date_picker/_date_picker.scss

```diff
@@ -0,0 +1,1018 @@
+    }
+    // Month-Year Select Styles
+    select.flatpickr-monthDropdown-months {
+      font-size: 16px;
+      font-weight: 600;
+      color: #242B42;
+      margin-top: 0px;
+      margin-bottom: 0px;
+    }
+
+    select.numInput {
+      font-size: 16px;
+      font-weight: 600;
+      color: #242B42;
+      border: 0px;
+      font-family: inherit;
+    }
+
+    // Calendar Shadow
+    .flatpickr-calendar {
+      box-shadow: $shadow_deeper;
+    }
+
+    // Day and Range Styles
+    .flatpickr-day {
+      user-select: none;
+      &:hover {
+        background-color: rgba($color: $neutral, $alpha: 0.2);
+        border-color: rgba($color: $neutral, $alpha: 0.2);
+        box-shadow: $shadow_deep;
+      }
+    }
+    
+    .flatpickr-day.today {
+      font-weight: 600;
+      color: $charcoal;
+      &.selected {
+        color: $text_dk_default;
+      }
+    }
+    .flatpickr-day.selected, .flatpickr-day.startRange, .flatpickr-day.endRange {
+      background-color: $primary;
+      border-color: $primary;
+      box-shadow: $shadow_deep;
+      font-weight: 600;
+    }
+    // had to use flat hex colors without alpha channels
+    // the shadows were overlapping and opacities were adding together in spots
+    .flatpickr-day.inRange {
+      background-color: #E5EEFA;
+      border: 2px;
+      box-shadow: -5px 0 0 #E5EEFA, 5px 0 0 #E5EEFA;
+    }
+  }
+
+  &.error {
+    [class^=pb_body_kit] {
+      margin-top: $space_xs / 2;
+    }
+    .flatpickr-wrapper {
+      > input:first-child {
+        border-color: $error;
+      }
+    }
+  }
+}
+
+.pb_popover_body {
+  overflow: visible !important;
+}
+
+// manual import -- flatpickr styles
+
+.flatpickr-calendar {
+  background: transparent;
+  opacity: 0;
+  display: none;
+  text-align: center;
+  visibility: hidden;
+  padding: 0;
+  -webkit-animation: none;
+          animation: none;
+  direction: ltr;
+  border: 0;
+  font-size: 14px;
+  line-height: 24px;
+  border-radius: 5px;
+  position: absolute;
+  width: 307.875px;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  -ms-touch-action: manipulation;
+      touch-action: manipulation;
+  background: #fff;
+  -webkit-box-shadow: 1px 0 0 #e6e6e6, -1px 0 0 #e6e6e6, 0 1px 0 #e6e6e6, 0 -1px 0 #e6e6e6, 0 3px 13px rgba(0,0,0,0.08);
+          box-shadow: 1px 0 0 #e6e6e6, -1px 0 0 #e6e6e6, 0 1px 0 #e6e6e6, 0 -1px 0 #e6e6e6, 0 3px 13px rgba(0,0,0,0.08);
+}
+.flatpickr-calendar.open,
+.flatpickr-calendar.inline {
+  opacity: 1;
+  max-height: 640px;
+  visibility: visible;
+}
+.flatpickr-calendar.open {
+  display: inline-block;
+  z-index: 99999;
+}
+.flatpickr-calendar.animate.open {
+  -webkit-animation: fpFadeInDown 300ms cubic-bezier(0.23, 1, 0.32, 1);
+          animation: fpFadeInDown 300ms cubic-bezier(0.23, 1, 0.32, 1);
+}
+.flatpickr-calendar.inline {
+  display: block;
+  position: relative;
+  top: 2px;
+}
+.flatpickr-calendar.static {
+  position: absolute;
+  top: calc(100% + 2px);
```

Comment:
Could this cause a problem in certain layouts? Are you expecting 100% to be the width of the screen or the parent container?